py3status/modules/xrandr.py

```diff
@@ -384,6 +411,7 @@ def xrandr(self):
             full_text = self._center(self.displayed)
         else:
             full_text = self.displayed
+        full_text = self.py3.safe_format(self.format, {'xrandr': full_text})
```

Comment:
Thanks for catching that. We had github major outage two hours ago. Could add cycling github status with that info / color / message. This should be finished?